Mail2Bug/Config.cs

```diff
@@ -263,6 +269,8 @@ public System.Security.Cryptography.DataProtectionScope EncryptionScope
                 }
             }
 
+            public bool ConvertInlineAttachments { get; set; }
```

Comment:
Let's add this to the fully annotated config example with a description of what it does, and mention that the default is 'true'